section3/reflection.md

```diff
@@ -1,13 +1,29 @@
 ## Section 3 Reflection
 
 1. In Section 3, we removed even more supports and pushed you to learn a bit more independently, read through technical documentation, and use your resources more. How did that feel?
+- Since I had already done some trouble-shooting in section 2, I felt more comfortable with how I was pushed in section 3. I still feel like I need to read the technical documentation at least twice to get it, but I still feel like I eventually get it.
+
+---
 
 2. Think of a web or mobile application you use regularly. Where is an example of a place that the developers probably use Arrays? Explain.
+- I use Pinterest often for recipes, and I think when listing the ingredients they use arrays, because the list shows up in different places and they can just use the array instead of entering each ingredient every time.
+
+---
 
 3. In your own words, explain what an Array method is.
+- An array is a way to store more than one pice of info in a variable. For example, if your var was a jewelry box, instead of just saying inside are rings, you could list all of the rings in the jewelry box.
+
+---
 
 4. What is the purpose of a loop? When we would use a loop in conjunction with array vs. without?
+-Loops are a way of performing an action on a list of items, with one set of code, instead of writing new code to perform the action on each item. It is helpful to use a loop with an array when we want to add a statement to each item. This will save us from typing the action out every time for each item. It is helpful to use loops without an array if we just want the same thing to run over and over again. For example, if we wanted clap to run to make it look like the clapping was repeating, we would use the loop to specify how many times we want it to run, but just run that one word.
+
+---
 
 5. If you had to teach a 5 year old about loops, how would you explain it?
+- A loop is a way to run a set of HotWheels through the racetrack without pushing each one individually. You can line them all up, and have something push the whole line.
+
+---
 
 6. What questions do you still have about Arrays and loops?
+- I still have questions about placements of brackets, when combining more than one loop or action within the loop. I also know I just need more practice with the functions. 
```

Comment:
Ok - is this referring to your voting ages problem maybe? I made a comment about it too! If it is, let me know that's what you need help with and I can send you what that should look like.